### PR TITLE
Create make_api_call.js

### PR DESCRIPTION
**Pull request by sazzad-eu**
Add sample code to make api call from action

### CODE REVIEW

**Comment by hifabienne** [comment]
Thanks for moving the example to this repo and for your contribution.
@livio-a @adlerhurst can one of you have a look at it?

File: examples/make_api_call.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+let http = require('zitadel/http')
```
Comment (livio-a) [inline examples/make_api_call.js:1]:
```suggestion
/**
 * Call an HTTP endpoint to send / retrieve information.
 * This example show a simple POST method and logs the retrieved id.
 *
 * Flow: External/Internal Authentication and Complement token, Trigger: all
 *
 * @param ctx
 * @param api
 */
let http = require('zitadel/http')
```

**Review by livio-a — COMMENTED** [review]
hey @sazzad-eu thanks for the PR. I'd just add a short header

**Comment by sazzad-eu** [comment]
> hey @sazzad-eu thanks for the PR. I'd just add a short header

Committed suggestion. Thank you

**Review by livio-a — APPROVED** [review]

**Pull request merged**